stable/traefik/templates/configmap.yaml

```diff
@@ -37,7 +37,7 @@ data:
       address = ":443"
       {{- if .Values.proxyProtocol.enabled }}
         [entryPoints.https.proxyProtocol]
-        {{ template "proxyprotocol" . }}
+        {{ template "trustedips" . }}
```

Comment:
@timstoop, I'm wondering now why this uses a template at all. This is only used in one spot, isn't it?

Reply:
Because of this: https://github.com/kubernetes/charts/pull/2824#pullrequestreview-81993454

It's in two places, actually.

Reply:
I remembered making that comment, but couldn't spot the two places this morning. I see them now.